styles.css

```diff
@@ -37,67 +37,108 @@ body {
 }
 
 header {
-  width: calc(100% - 40px);
+  width: calc(100% - 6px);
   font-weight: 600;
   font-size: 1.3rem;
-  padding: 20px;
   display: flex;
   align-items: center;
   justify-content: space-between;
 }
 
-header .welcome-logo {
-  color: #333;
-  position: fixed;
-  z-index: 100;
-  top: 1rem;
-  left: 1rem;
-  cursor: pointer;
+.nav-branding {
+  color: black;
   text-decoration: none;
 }
 
-.menuItem {
-  display: block;
-  margin: 2rem 4rem;
-  font-size: 1.8rem;
-  color: rgb(236, 18, 18);
+.nav-branding.active {
+  color: white;
+}
+
+.navbar ul li {
+  list-style: none;
+}
+
+.navbar ul li a {
+  color: #4a3a3b;
+  font-size: 2rem;
+  font-weight: 600;
   text-decoration: none;
```

Comment:
- [Optional] Great Job for implementing the previous reviewer's suggestion 👍👌 I Think you missed something your mobile menu header is scrollable as you can see it on 👇 below video, Please fix it to have a better user interactivity 👍👍 

https://user-images.githubusercontent.com/52010789/217448389-88b21bbf-fb94-47cb-9170-a67b060b6426.mp4

Reply:
thank you @binhussen 🙏 ,
I will consider it.